src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Shared/LoginDisplay.IndividualLocalAuth.razor

```diff
@@ -26,9 +24,8 @@
 </AuthorizeView>
 
 @code{
-    private async Task BeginSignOut(MouseEventArgs args)
+    public void BeginSignOut()
```

Comment:
Why `public`?

Reply:
```suggestion
    private void BeginSignOut()
```

Reply:
No particular reason, I just made a mistake (it's sample code though). I have made it private